posthog/models/channel_type/sql.py

```diff
@@ -0,0 +1,59 @@
+import json
+import os
+
+from posthog.settings import CLICKHOUSE_CLUSTER
+
+CHANNEL_DEFINITION_TABLE_SQL = f"""
+CREATE TABLE IF NOT EXISTS channel_definition ON CLUSTER {CLICKHOUSE_CLUSTER} (
+    domain String NOT NULL,
+    kind String NOT NULL,
+    domain_type String NULL,
+    type_if_paid String NULL,
+    type_if_organic String NULL,
+) ENGINE = MergeTree()
+ORDER BY domain;
```

Comment:
I would sync the `ORDER BY` key for this table with the `PRIMARY KEY` of the dictionary below.
https://github.com/PostHog/posthog/pull/19085/files#diff-35d4b57e1a6f6bcfa90872805e15f38daaf02e2d0f99045832e3a08a50d6f5c0R51

Looks like it should be
```
ORDER BY domain, kind
```

Reply:
Done